tensorflow/core/kernels/gpu_utils.h

```diff
@@ -42,6 +42,10 @@ class AutotuneResult;
 
 namespace tensorflow {
 
+// returns true if bfloat16 is not directly supported in Ops and inputs shall be
+// casted to floats to perform the computations and then back
+bool IsBF16NotSupportedInOps(se::Stream *stream);
```

Comment:
Let's flip this so it's `IsBF16Supported` to avoid double-negatives when checking for support.